src/lib/finalizer.ts

```diff
@@ -26,14 +26,17 @@ export function addFinalizer<K extends KubernetesObject>(request: PeprMutateRequ
   }
 
   request.Merge({ metadata: { finalizers } } as DeepPartial<K>);
+
+  // Log the addition of a finalizer with the alias
+  Log.info({ alias: request.Request.name }, `Added finalizer '${peprFinal}' to ${request.Raw.metadata?.name}`);
```

Comment:
Does this provide value to the user? We are trying to cut our logs down
https://github.com/defenseunicorns/pepr/issues/1360

Reply:
Removed the log message